floats.go

```diff
@@ -69,9 +69,8 @@ func AddScaledTo(dst, y []float64, alpha float64, s []float64) []float64 {
 	if len(dst) != len(s) || len(dst) != len(y) {
 		panic("floats: lengths of slices do not match")
 	}
-	for i, val := range s {
-		dst[i] = y[i] + alpha*val
-	}
+	copy(dst, y)
```

Comment:
There's no performance penalty in the assembly version. I haven't checked the pure Go version, where another bound check is introduced in z[i] when you convert
```y[i] += alpha * v``` to ```z[i] = alpha * v + y[i]```.